components/CaseStudiesProject.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card } from "@nextui-org/card";
+import { Link } from "@nextui-org/link";
+import Image from "next/image";
+
+const CaseStudiesProject = ({
+  project,
+}: {
+  project: { name: string; description: string; image: string; color: string };
+}) => {
+  return (
+    <>
+      <Card
+        as={Link}
+        href={`/case-studies/${project.name.toLowerCase()}`}
+        className={`${project.color} items-center rounded-[0.62725rem] px-[1.54rem] pt-[2.73rem] xl:rounded-2xl xl:px-[3.75rem] xl:pt-[4.35rem]`}
+      >
+        <Image
+          className="lg:h-[15.8rem] lg:w-[26.8rem]"
+          src={project.image}
+          width={278.486}
+          height={165.054}
```

Comment:
FYI: The width and height attributes are used to infer the correct aspect ratio of the image and avoid layout shift from the image loading in. The width and height do not determine the rendered size of the image file. Learn more about [Image Sizing](https://nextjs.org/docs/app/building-your-application/optimizing/images#image-sizing).
You can also read more about it here: https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images